### PR TITLE
more features

### PR DESCRIPTION
**Pull request by DeveloperOl**
Added some small design changes to fit native clients and use persistent redux storage to save connection data.

### CODE REVIEW

**Comment by DrPandemic** [comment]
Could you write a simple description please? This will help us test/review. Or if the PR is not ready, mark it as WIP

**Comment by DeveloperOl** [comment]
@DrPandemic Updated. This whole project is still a WIP tho 😆

**Comment by DeveloperOl** [comment]
There is still a lot to do, but this should be merged to keep the master up2date for other forks.

File: jellyfinclient/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "formik": "^1.4.2",
     "fs-extra": "7.0.0",
     "is-electron": "^2.2.0",
-    "jellyfin-apiclient": "https://github.com/jellyfin/jellyfin-apiclient-javascript.git",
+    "jellyfin-apiclient": "^1.0.0",
```
Comment (EraYaN) [inline jellyfinclient/package.json:39]: Is there a reason for this change? Did we release the api-client?

File: jellyfinclient/src/utilities/storage/store.js [hunk 2]
```diff
@@ -18,21 +17,21 @@ const rootPersistConfig = {
...
 const authPersistConfig = {
-    key: 'auth',
+    key: 'authCredentials',
     storage: sensitiveStorage,
     stateReconciler: hardSet,
 }
 
 const connectPersistConfig = {
-    key: 'connect',
+    key: 'connectionStatus',
```
Comment (EraYaN) [inline jellyfinclient/src/utilities/storage/store.js:26]: Shouldn't these keys mirror the keys in the combined reducer below?

**Review by EraYaN — COMMENTED** [review]
LGTM, some small questions.

File: jellyfinclient/package.json (hunk 1 above)
Comment (DeveloperOl) [reply jellyfinclient/package.json:39]: Yes the prebuild packages are now available from npmjs by @anthonylavado

File: jellyfinclient/src/utilities/storage/store.js (hunk 2 above)
Comment (DeveloperOl) [reply jellyfinclient/src/utilities/storage/store.js:26]: Yes I missed that. Updated in e095478e5d3b0fec561443ac2b070a3244a00e50

**Review by EraYaN — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**